taskcluster/kinds/bicleaner/kind.yml

```diff
@@ -80,7 +80,10 @@ tasks:
                 COMPRESSION_CMD: zstdmt
                 ARTIFACT_EXT: zst
             # 128 happens when cloning this repository fails
-            retry-exit-status: [128]
+            # 9001 is the code for when tensorflow fails to find GPUs on the system,
+            # and biclenaer reverts to CPU time. Rather than waste time, we should
+            # restart the task.
+            retry-exit-status: [128,9001]
```

Comment:
Nice, I like the idea of targeted Taskcluster restarts based on specific exit codes. We can utilize this pattern for other tasks too instead of using restarts inside the Python code. It makes it more transparent to see when a task fails and restarts.